src/lib/model/WorkflowState.ts

```diff
@@ -0,0 +1,62 @@
+import { HalResource } from '../hal/models/HalResource';
+import { Page } from './Page';
+
+/**
+ * Workflow state
+ */
+
+export class WorkflowState extends HalResource {
+  /**
+   * Unique id generated on creation
+   */
+  public id: string;
+
+  /**
+   * RGB color of the workflow state
+   */
+  public color: string;
+
+  /**
+   * Id of the user responsible for originally creating the workflow state
+   */
+  public createdBy: string;
+
+  /**
+   * Timestamp representing when the workflow state was originally created in ISO 8601 format
+   */
+  public createdDate: string;
+
+  /**
+   * Friendly label for the workflow state
+   */
+  public label: string;
+
+  /**
+   * Id of the user responsible for the last update to the workflow state
+   */
+  public lastModifiedBy: string;
+
+  /**
+   * Timestamp representing when the workflow state was last updated in ISO 8601 format
+   */
+  public lastModifiedDate: string;
```

Comment:
Missing newline after

Reply:
Will fix

Reply:
This doesn't seem to have been resolved

Reply:
There's also a new misplaced newline between `related` and its doc.

Reply:
Oh, seems now I resolved this all